### ISSUE TITLE
magithub-issue-personal-note-file Wong type argument: listp, 4

### ISSUE THREAD

**Issue by wyuenho** [vermiculus/magithub#344]
```
Debugger entered--Lisp error: (wrong-type-argument listp 4)
  assq(owner 4)
  (cdr (assq 'owner (cdr (assq 'repo alist))))
  (assq 'login (cdr (assq 'owner (cdr (assq 'repo alist)))))
  (cdr (assq 'login (cdr (assq 'owner (cdr (assq 'repo alist))))))
  (let ((\.repo\.owner\.login (cdr (assq 'login (cdr (assq 'owner (cdr (assq 'repo alist))))))) (\.repo\.name (cdr (assq 'name (cdr (assq 'repo alist))))) (\.issue\.number (cdr (assq 'number (cdr (assq 'issue alist)))))) (expand-file-name (format "%s/%s/notes/%d.org" \.repo\.owner\.login \.repo\.name \.issue\.number) magithub-dir))
  (let ((alist (list (cons 'repo (magithub-repo (magithub-issue-repo issue-or-pr))) (cons 'issue issue-or-pr)))) (let ((\.repo\.owner\.login (cdr (assq 'login (cdr (assq 'owner (cdr (assq 'repo alist))))))) (\.repo\.name (cdr (assq 'name (cdr (assq 'repo alist))))) (\.issue\.number (cdr (assq 'number (cdr (assq 'issue alist)))))) (expand-file-name (format "%s/%s/notes/%d.org" \.repo\.owner\.login \.repo\.name \.issue\.number) magithub-dir)))
  magithub-issue-personal-note-file(((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")))
  magithub-issue-has-personal-note-p(((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")))
  magithub-issue-insert-section(((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")) 2)
  #f(compiled-function (it) #<bytecode 0x42117e59>)(((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")))
  mapcar(#f(compiled-function (it) #<bytecode 0x42117e59>) (((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/73") (id . 322567505) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg3NjU0MjEz") (number . 73) (title . "Remove rjsx-record-tag-bracket-face used before defined compiler warning") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-13T06:03:46Z") (updated_at . "2018-06-07T04:29:18Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/73") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/73") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/73.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/73.patch")) (body . "`rjsx-record-tag-bracket-face` is used in a number of functions before its definition. Not that it matters in practice, but emacs's byte compiler is too stupid to notice that, might as well get rid of that warning anyway.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/72") (id . 322558614) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg3NjQ5MzU4") (number . 72) (title . "Fix #46 Support JSX comments") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 4) (created_at . "2018-05-13T02:36:38Z") (updated_at . "2018-06-15T22:51:54Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/72") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/72") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/72.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/72.patch")) (body . "This PR implements a number of comment commands suitable for `rjsx-mode`. \015\n\015\n`rjsx-comment-dwim` generally does what you expect - tries really hard to keep your JSX code valid by deciding what kind of `comment-start` and `comment-end` delimiters to use. If point is on a top-level JSX element, it'll just comment the whole node with `/* */`. If point is on a child element or text node, the child is commented with `{/* */}`. If point is on or inside a JSX expression, the body of the JSX expression is commented with `/* */`. If point  is on or before a JSX attribute, the entire attribute is commented with `/* */`.\015\n\015\nThis PR also provides an implementation of `comment-region-function`, `uncomment-region-function` and `comment-quote-nested-function` to support `rjsx-comment-dwim`.\015\n\015\nThere's a shit ton of tests included. Generally, `rjsx-comment-dwim` will just do what you expect. In the few edge cases that it doesn't, you can cycle the comments delimiters until you get what you want, or by moving point one character before or after its current position before `M-;`\015\n\015\nI think this PR is so good, I've remapped `comment-dwim` to `rjsx-comment-dwim` when `rjsx-mode` is activated.\015\n\015\nP.S. This is some of the ugliest and nastiest code I've ever written, and it's by no means easy as there are a LOT of cases to take care of and involves quite a number of regular expression hacks. I really hate `newcomment` and these stupid `comment-` global variables.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/66") (id . 310260517) (node_id . "MDExOlB1bGxSZXF1ZXN0MTc4NjkzOTcy") (number . 66) (title . "provide rjsx-minor-mode") (user (login . "axyz") (id . 412723) (node_id . "MDQ6VXNlcjQxMjcyMw==") (avatar_url . "https://avatars2.githubusercontent.com/u/412723?v=4") (gravatar_id . "") (url . "https://api.github.com/users/axyz") (html_url . "https://github.com/axyz") (followers_url . "https://api.github.com/users/axyz/followers") (following_url . "https://api.github.com/users/axyz/following{/other_user}") (gists_url . "https://api.github.com/users/axyz/gists{/gist_id}") (starred_url . "https://api.github.com/users/axyz/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/axyz/subscriptions") (organizations_url . "https://api.github.com/users/axyz/orgs") (repos_url . "https://api.github.com/users/axyz/repos") (events_url . "https://api.github.com/users/axyz/events{/privacy}") (received_events_url . "https://api.github.com/users/axyz/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 13) (created_at . "2018-03-31T18:15:37Z") (updated_at . "2018-05-14T20:00:45Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/66") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/66") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/66.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/66.patch")) (body . "I'm not sure if this is the cleanest way to do it or if there may be unexpected side effect.\015\nThe idea is to provide a minor-mode similar to js2-minor-mode in order to allow the usage of packages depending on the js2/rjsx AST to work in other major modes (e.g. using js2-refactor in web-mode)\015\n\015\nI tired it locally and seems to work with js2-refactor")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/49") (id . 261237072) (node_id . "MDExOlB1bGxSZXF1ZXN0MTQzNTg3MDgx") (number . 49) (title . "Vanish tag") (user (login . "felipeochoa") (id . 2739312) (node_id . "MDQ6VXNlcjI3MzkzMTI=") (avatar_url . "https://avatars1.githubusercontent.com/u/2739312?v=4") (gravatar_id . "") (url . "https://api.github.com/users/felipeochoa") (html_url . "https://github.com/felipeochoa") (followers_url . "https://api.github.com/users/felipeochoa/followers") (following_url . "https://api.github.com/users/felipeochoa/following{/other_user}") (gists_url . "https://api.github.com/users/felipeochoa/gists{/gist_id}") (starred_url . "https://api.github.com/users/felipeochoa/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/felipeochoa/subscriptions") (organizations_url . "https://api.github.com/users/felipeochoa/orgs") (repos_url . "https://api.github.com/users/felipeochoa/repos") (events_url . "https://api.github.com/users/felipeochoa/events{/privacy}") (received_events_url . "https://api.github.com/users/felipeochoa/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2017-09-28T08:39:36Z") (updated_at . "2017-10-10T11:34:05Z") (closed_at) (author_association . "OWNER") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/49") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/49") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/49.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/49.patch")) (body . "Especially with React 16, it would be nice to have a single command to unwrap the children of a tag and splice them into the surroundings:\015\n\015\n```jsx\015\nlet c = <Parent><Child/><Parent>; // point on Parent\015\n```\015\n\015\nafter vanishing parent should give:\015\n\015\n```jsx\015\nlet c = <Child/>; // point right before Child\015\n```\015\n\015\n\015\n- [X] Initial implementation\015\n- [ ] Clean up whitespace outside the removed tag (e.g., extra newlines)\015\n- [ ] Clean up whitespace when first/last child are text nodes\015\n- [ ] Fix indentation for children\015\n- [ ] If only child is text, wrap it in quotes\015\n- [ ] Set point and mark to the start/end of the child region (or reversed if point was on the closer)\015\n- [ ] Add tests"))))
  magithub-issue-insert-sections((((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/75/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (id . 323453742) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg4MzA3OTI5") (number . 75) (title . "Indentation Fixes") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-16T03:10:25Z") (updated_at . "2018-05-16T23:13:19Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/75") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/75") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/75.patch")) (body . "# Merge after #72 \015\n\015\n## [Diff](https://github.com/wyuenho/rjsx-mode/compare/jsx-comments...wyuenho:indent-after-jsx-expr)\015\n\015\nFixes:\015\n- mooz/js2-mode#490\015\n- mooz/js2-mode#482\015\n- mooz/js2-mode#462\015\n- mooz/js2-mode#451\015\n\015\nSimilar to `js-jsx-indent-line`, but fixes indentation bug with JSXElement after a JSX expression and arrow function. In addition, the > of a multi-line open tag and the /> of a multi-line self-closing tag are aligned to the beginning <. This function also ensures everything inside a JSX context is indented according to `js-indent-level` using spaces, this is due to the limitation of `sgml-indent-line` not being able to indent with tabs.\015\n\015\nAs an added bonus, this also fixes the `rjsx-indentation-1` test for Emacs < 26.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/73/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/73") (id . 322567505) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg3NjU0MjEz") (number . 73) (title . "Remove rjsx-record-tag-bracket-face used before defined compiler warning") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2018-05-13T06:03:46Z") (updated_at . "2018-06-07T04:29:18Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/73") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/73") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/73.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/73.patch")) (body . "`rjsx-record-tag-bracket-face` is used in a number of functions before its definition. Not that it matters in practice, but emacs's byte compiler is too stupid to notice that, might as well get rid of that warning anyway.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/72/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/72") (id . 322558614) (node_id . "MDExOlB1bGxSZXF1ZXN0MTg3NjQ5MzU4") (number . 72) (title . "Fix #46 Support JSX comments") (user (login . "wyuenho") (id . 160028) (node_id . "MDQ6VXNlcjE2MDAyOA==") (avatar_url . "https://avatars1.githubusercontent.com/u/160028?v=4") (gravatar_id . "") (url . "https://api.github.com/users/wyuenho") (html_url . "https://github.com/wyuenho") (followers_url . "https://api.github.com/users/wyuenho/followers") (following_url . "https://api.github.com/users/wyuenho/following{/other_user}") (gists_url . "https://api.github.com/users/wyuenho/gists{/gist_id}") (starred_url . "https://api.github.com/users/wyuenho/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/wyuenho/subscriptions") (organizations_url . "https://api.github.com/users/wyuenho/orgs") (repos_url . "https://api.github.com/users/wyuenho/repos") (events_url . "https://api.github.com/users/wyuenho/events{/privacy}") (received_events_url . "https://api.github.com/users/wyuenho/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 4) (created_at . "2018-05-13T02:36:38Z") (updated_at . "2018-06-15T22:51:54Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/72") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/72") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/72.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/72.patch")) (body . "This PR implements a number of comment commands suitable for `rjsx-mode`. \015\n\015\n`rjsx-comment-dwim` generally does what you expect - tries really hard to keep your JSX code valid by deciding what kind of `comment-start` and `comment-end` delimiters to use. If point is on a top-level JSX element, it'll just comment the whole node with `/* */`. If point is on a child element or text node, the child is commented with `{/* */}`. If point is on or inside a JSX expression, the body of the JSX expression is commented with `/* */`. If point  is on or before a JSX attribute, the entire attribute is commented with `/* */`.\015\n\015\nThis PR also provides an implementation of `comment-region-function`, `uncomment-region-function` and `comment-quote-nested-function` to support `rjsx-comment-dwim`.\015\n\015\nThere's a shit ton of tests included. Generally, `rjsx-comment-dwim` will just do what you expect. In the few edge cases that it doesn't, you can cycle the comments delimiters until you get what you want, or by moving point one character before or after its current position before `M-;`\015\n\015\nI think this PR is so good, I've remapped `comment-dwim` to `rjsx-comment-dwim` when `rjsx-mode` is activated.\015\n\015\nP.S. This is some of the ugliest and nastiest code I've ever written, and it's by no means easy as there are a LOT of cases to take care of and involves quite a number of regular expression hacks. I really hate `newcomment` and these stupid `comment-` global variables.")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/66/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/66") (id . 310260517) (node_id . "MDExOlB1bGxSZXF1ZXN0MTc4NjkzOTcy") (number . 66) (title . "provide rjsx-minor-mode") (user (login . "axyz") (id . 412723) (node_id . "MDQ6VXNlcjQxMjcyMw==") (avatar_url . "https://avatars2.githubusercontent.com/u/412723?v=4") (gravatar_id . "") (url . "https://api.github.com/users/axyz") (html_url . "https://github.com/axyz") (followers_url . "https://api.github.com/users/axyz/followers") (following_url . "https://api.github.com/users/axyz/following{/other_user}") (gists_url . "https://api.github.com/users/axyz/gists{/gist_id}") (starred_url . "https://api.github.com/users/axyz/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/axyz/subscriptions") (organizations_url . "https://api.github.com/users/axyz/orgs") (repos_url . "https://api.github.com/users/axyz/repos") (events_url . "https://api.github.com/users/axyz/events{/privacy}") (received_events_url . "https://api.github.com/users/axyz/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 13) (created_at . "2018-03-31T18:15:37Z") (updated_at . "2018-05-14T20:00:45Z") (closed_at) (author_association . "NONE") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/66") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/66") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/66.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/66.patch")) (body . "I'm not sure if this is the cleanest way to do it or if there may be unexpected side effect.\015\nThe idea is to provide a minor-mode similar to js2-minor-mode in order to allow the usage of packages depending on the js2/rjsx AST to work in other major modes (e.g. using js2-refactor in web-mode)\015\n\015\nI tired it locally and seems to work with js2-refactor")) ((url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49") (repository_url . "https://api.github.com/repos/felipeochoa/rjsx-mode") (labels_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/labels{/name}") (comments_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/comments") (events_url . "https://api.github.com/repos/felipeochoa/rjsx-mode/issues/49/events") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/49") (id . 261237072) (node_id . "MDExOlB1bGxSZXF1ZXN0MTQzNTg3MDgx") (number . 49) (title . "Vanish tag") (user (login . "felipeochoa") (id . 2739312) (node_id . "MDQ6VXNlcjI3MzkzMTI=") (avatar_url . "https://avatars1.githubusercontent.com/u/2739312?v=4") (gravatar_id . "") (url . "https://api.github.com/users/felipeochoa") (html_url . "https://github.com/felipeochoa") (followers_url . "https://api.github.com/users/felipeochoa/followers") (following_url . "https://api.github.com/users/felipeochoa/following{/other_user}") (gists_url . "https://api.github.com/users/felipeochoa/gists{/gist_id}") (starred_url . "https://api.github.com/users/felipeochoa/starred{/owner}{/repo}") (subscriptions_url . "https://api.github.com/users/felipeochoa/subscriptions") (organizations_url . "https://api.github.com/users/felipeochoa/orgs") (repos_url . "https://api.github.com/users/felipeochoa/repos") (events_url . "https://api.github.com/users/felipeochoa/events{/privacy}") (received_events_url . "https://api.github.com/users/felipeochoa/received_events") (type . "User") (site_admin)) (labels) (state . "open") (locked) (assignee) (assignees) (milestone) (comments . 0) (created_at . "2017-09-28T08:39:36Z") (updated_at . "2017-10-10T11:34:05Z") (closed_at) (author_association . "OWNER") (pull_request (url . "https://api.github.com/repos/felipeochoa/rjsx-mode/pulls/49") (html_url . "https://github.com/felipeochoa/rjsx-mode/pull/49") (diff_url . "https://github.com/felipeochoa/rjsx-mode/pull/49.diff") (patch_url . "https://github.com/felipeochoa/rjsx-mode/pull/49.patch")) (body . "Especially with React 16, it would be nice to have a single command to unwrap the children of a tag and splice them into the surroundings:\015\n\015\n```jsx\015\nlet c = <Parent><Child/><Parent>; // point on Parent\015\n```\015\n\015\nafter vanishing parent should give:\015\n\015\n```jsx\015\nlet c = <Child/>; // point right before Child\015\n```\015\n\015\n\015\n- [X] Initial implementation\015\n- [ ] Clean up whitespace outside the removed tag (e.g., extra newlines)\015\n- [ ] Clean up whitespace when first/last child are text nodes\015\n- [ ] Fix indentation for children\015\n- [ ] If only child is text, wrap it in quotes\015\n- [ ] Set point and mark to the start/end of the child region (or reversed if point was on the closer)\015\n- [ ] Add tests"))))
  magithub-issue--insert-pr-section()
  run-hooks(magit-status-sections-hook)
  magit-run-section-hook(magit-status-sections-hook)
  magit-status-refresh-buffer()
  apply(magit-status-refresh-buffer nil)
  magit-refresh-buffer()
  magit-refresh()
  magit-run-git("remote" "set-head" "origin" "--delete")
  magit-remote-unset-head("origin")
  #f(compiled-function (process event) #<bytecode 0x61951745>)(#<process git> "finished\n")
  apply(#f(compiled-function (process event) #<bytecode 0x61951745>) (#<process git> "finished\n"))
  #f(advice-wrapper :after #f(compiled-function (process event) #<bytecode 0x61951745>) #f(compiled-function (process event) #<bytecode 0x619517a1>))(#<process git> "finished\n")
```

**Comment by am** [comment]
I have the same problem. Not sure if it relates to this, but I cannot update the pull-request list on Magit status buffer.
